journals/apps/journals/management/commands/gather_videos.py

```diff
@@ -52,6 +52,22 @@ def get_video_course_runs_for_site(self, site):
         )
         return course_runs
 
+    def get_collection_for_site(self, site):
+        """
+        Args:
+            site: site object
+
+        Returns: Collection associated with given site
+        """
+        site_collection_name = site.site_name
+        collection = Collection.objects.filter(name=site_collection_name).first()
+        if not collection:
+            collection = Collection(name=site_collection_name)
+            root_collection = Collection.get_first_root_node()
+            root_collection.add_child(instance=collection)
```

Comment:
would the following be needed:
```
collection.save()
```

Reply:
@ziafazal 
please also update the create_sites.py management command to add video permissions in there as it already does for images and documents

Reply:
We don't need this now, but in the future we might consider creating a sub-collection beneath the site collection for the Course so that videos can be easily segmented by course. For example:
MITxPro->Course a
MITxPro->Course b
Don't need this now but maybe in the future

Reply:
@bill-filler I was not able to find any `create_sites.py` can you paste a link to it here?
About collections this is the way i found to create a collection from code. It appears every collection needs to have a root collection.

Reply:
@ziafazal 
https://github.com/edx/journals/blob/master/journals/apps/core/management/commands/create_site.py

Reply:
@ziafazal 
See the create_collection() method for what I meant about calling .save()
https://github.com/edx/journals/blob/master/journals/apps/core/management/commands/create_site.py#L201
not sure if it's necessary but we should be consistent

Reply:
@bill-filler ` .save()` is not needed.